.github/workflows/semantic_release.yml

```diff
@@ -0,0 +1,70 @@
+# This workflow runs semantic release, bumps, generates changelog, and tags the project
+name: Semantic Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: "Are you releasing a new version? (Y/n)"
+        required: true
+        type: string
+
+jobs:
+  semantic-release:
+    if: contains('["seebees","texastony","ShubhamChaturvedi7","lucasmcdonald3","josecorella","imabhichow","rishav-karanjit","antonf-amzn","justplaz","ajewellamz"]', github.actor)
```

Comment:
Is it worth considering something like https://github.com/tspascoal/get-user-teams-membership?

Reply:
that would be nice but unfortunately it doesn't look like it is a verified gha; otherwise I would use it

Reply:
Make sense.
Can we add a comment that points to the team this is expected to be synced with here?